lib/mix/tasks/generate_spec_tests.ex

```diff
@@ -59,16 +59,19 @@ defmodule Mix.Tasks.GenerateSpecTests do
     module_name = "Elixir.#{c}.#{f}.#{r}Test" |> String.to_atom()
     runner_module = "Elixir.#{r}TestRunner" |> String.to_atom()
 
+    # TODO: we can isolate tests that use the DB from each other by using ExUnit's tmp_dir context option.
     header = """
     defmodule #{module_name} do
       use ExUnit.Case, async: false
 
       setup_all do
+        start_link_supervised!({LambdaEthereumConsensus.Store.Db, db_location: "test/generated/#{config}_#{fork}_#{runner}_test_db"})
```

Comment:
This would prevent tests to run in parallel right? Since we would try to spawn more than one genserver with the same name? (LambdaEthereumConsensus.Store.Db)

Reply:
Exactly! I'll try to start this only once in `test_helper.exs` in another PR